### ISSUE TITLE
Add state.matchesAny("state")

### ISSUE THREAD

**Issue by SurajThotakura** [akhil0001/simple-state-machine#32]
Add state.matches("state") util to check current state

**Comment by akhil0001** [comment]
Since we dont have compound states support in this lib, `.matches` would essentially make a string equality check.

**Comment by akhil0001** [comment]
Will add this in upcoming package update

**Comment by akhil0001** [comment]
@SurajThotakura  added `matchesAny` function to state. 

Usage:
```js
const {state, send} = useMachine(exampleMachine);

const isLoading = state.matchesAny('fetching', 'buffering');
```

`isLoading` will be true if state matches anyone of the passed strings. And this `matchesAny` is typesafe

This is available from `0.1.0-alpha.3` version of the packages.
